packages/app/src/components/drawer/accounts/create-account.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useEffect } from 'react';
+import { Box, Flex, Text, Button } from '@stacks/ui';
+import { useWallet } from '@common/hooks/use-wallet';
+
+interface CreateAccountProps {
+  close: () => void;
+}
+export const CreateAccount: React.FC<CreateAccountProps> = ({ close }) => {
+  const { doCreateNewAccount } = useWallet();
+
+  const createAccount = useCallback(async () => {
+    await doCreateNewAccount();
+    setTimeout(() => close(), 3000);
```

Comment:
```
export function delay(delayMs: number) {
  return new Promise(resolve => setTimeout(() => resolve(), delayMs));
}
```
super handy

Reply:
Hah I do have that in our test suite - in this case I'd rather just keep it light, and I don't need to `await` here